core/execution/engine.go

```diff
@@ -721,23 +721,7 @@ func (e *Engine) submitSpotMarket(ctx context.Context, marketConfig *types.Marke
 		)
 		return err
 	}
-	mkt, err := spot.NewMarket(
-		ctx,
-		e.log,
-		e.Matching,
-		e.Fee,
-		e.Liquidity,
-		e.collateral,
-		marketConfig,
-		e.timeService,
-		e.broker,
-		mas,
-		e.stateVarEngine,
-		e.marketActivityTracker,
-		bad,
-		qad,
-		e.peggedOrderCountUpdated,
-	)
+	mkt, err := spot.NewMarket(e.log, e.Matching, e.Fee, e.Liquidity, e.collateral, marketConfig, e.timeService, e.broker, mas, e.stateVarEngine, e.marketActivityTracker, bad, qad, e.peggedOrderCountUpdated)
```

Comment:
is there actually a change here other than formatting (which I like on the left more)?

Reply:
The context was not needed, I can reformat to what is was. It will be clearer